### ISSUE TITLE
No response

### ISSUE THREAD

**Issue by HireshPillay** [udacity/ud843_Soonami#20]
I am not able to see any response on the soonami app

**Comment by harshbajpai27** [comment]
Yes, i am facing the same problem, no data is being displayed on screen
![Capture](https://user-images.githubusercontent.com/31615743/69417238-45e52500-0d3e-11ea-840b-eb72bd2c3069.PNG)

**Comment by Moustafaali2019** [comment]
same as you but i have this exception reported:
2019-12-06 13:59:21.755 3867-3930/com.example.android.soonami E/MainActivity: Problem retrieving the earthquake JSON results.
    **java.net.SocketException: socket failed: EPERM (Operation not permitted)**

**Comment by LaszloLajosT** [comment]
> I am not able to see any response on the soonami app

Hi,

Did you sync the Grandle files and added the internet permission into Manifest file? 
This line of code needed to add:
`<uses-permission android:name="android.permission.INTERNET" />`

Plus, I needed to upgrade what Android studio said during my first application run.

Additional info:
I used this code inside MainActivity.java:
```
/** URL to query the USGS dataset for earthquake information */
    private static final String USGS_REQUEST_URL =
            "https://earthquake.usgs.gov/fdsnws/event/1/query?format=geojson&starttime=2014-01-01&endtime=2014-12-01&minmagnitude=7";
    
```

**Comment by Noman5237** [comment]
On my google pixel 3A api 29 uninstalling the app first before reinstalling again works
